web-ui/view/documentation.ml

```diff
@@ -0,0 +1,90 @@
+        [
+          div
+            ~a:[ a_class [ "blog-sidebar" ] ]
+            [
+              div
+                ~a:[ Tyxml_helpers.x_data "{submenu: false}" ]
+                [ (* this is where we would have the sidebar links if we had any *) ];
+            ];
+          div
+            ~a:[ a_class [ "prose col-span-4" ] ]
+            [
+              h1 [ txt "Getting Started" ];
+              p [ txt "OCaml-CI is a CI service for OCaml projects." ];
+              hr ();
+              p
+                [
+                  txt
+                    "It uses metadata from the project's opam and dune files \
+                     to work out what to build, and uses caching to make \
+                     builds fast. It automatically tests projects against \
+                     multiple OCaml versions and OS platforms.";
+                ];
+              p [ txt "To start building with OCaml-CI:" ];
+              ul
+                [
+                  li
+                    [
+                      txt "Add the app to your account, selecting ";
+                      code ~a:[ a_class [ "notranslate" ] ] [ txt "Install" ];
+                      txt " at ";
+                      a
+                        ~a:[ a_href "https://github.com/apps/ocaml-ci" ]
+                        [ txt "https://github.com/apps/ocaml-ci" ];
+                    ];
+                  li
+                    [
+                      txt
+                        "Select only the repositories you want tested - if you \
+                         select ";
+                      code [ txt "All Repositories" ];
+                      txt " we won't build anything!";
+                    ];
+                ];
+              p
+                [
+                  txt
+                    "This service is somewhat experimental and being heavily \
+                     worked on under ";
+                  a
+                    ~a:[ a_href "https://github.com/ocurrent" ]
+                    [ txt "https://github.com/ocurrent" ];
+                  txt
+                    ". You will need to be approved before it will start \
+                     building anything. e.g. ";
+                  a
+                    ~a:
+                      [ a_href "https://github.com/ocurrent/ocaml-ci/pull/346" ]
+                    [ txt "https://github.com/ocurrent/ocaml-ci/pull/346" ];
+                ];
+              ul
+                [
+                  li
+                    [
+                      txt "Github documentation on how to install ";
+                      a
+                        ~a:
+                          [
+                            a_href
+                              "https://docs.github.com/en/developers/apps/managing-github-apps/installing-github-apps#installing-your-private-github-app-on-your-repository";
+                          ]
+                        [ txt "an app on your repository" ];
```

Comment:
Instead, shouldn't we point to the dev.md on ocurrent.org? or the one on GitHub?

Reply:
I don't understand this suggestion - could you please provide a link?

Reply:
I was thinking about this [page](https://github.com/ocurrent/ocaml-ci/blob/master/doc/dev.md) and in a near future, it's [ocurrent.org version](https://www.ocurrent.org/docs/ocurrent/ocurrent/)

Reply:
@maiste I don't think this page is relevant to the users of ocaml-ci - rather the developers of ocaml-ci. Perhaps I am missing something in your suggestion?

Reply:
The part I misunderstood is the fact that the document explained how to install the GitHub App, not create your own. All good about it. Closing :)